### DISCUSSION TITLE
To force the refresh of the page

### DISCUSSION THREAD

**Discussion by Execotryx** [epam/UUI#1383] [Q&A] [answered]
Hello. I hope that you are well.
I got a situation: I'm adding the new record in the database and I need the table to refresh to reflect the newly added item. Could you please advise how I can do that? I use the LazyDataSource with DataTable.

**Comment by AlekseyManetov** [comment, marked as answer]
Hi, do you have any signal that new item was added? In a nutshell you can force DataSource to reload items, just by passing the dependency to the `useLazyDataSource` hook, if this dependency change LazyDataSource will be recreated and new items will be loaded.
